src/job.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+import { logger } from "./index";
+
+export function job() {

```

Comment:
what is this function for?

Reply:
There's nothing there. But we could fill some jobs in it to run except the service, like prepare the database before runing the service.